src/main/java/uk/gov/hmcts/reform/prl/services/DraftAnOrderService.java

```diff
@@ -366,7 +370,7 @@ public Map<String, Object> populateDraftOrderDocument(CaseData caseData) {
         Map<String, Object> caseDataMap = new HashMap<>();
         DraftOrder selectedOrder = getSelectedDraftOrderDetails(caseData);
         caseDataMap.put("previewUploadedOrder", selectedOrder.getOrderDocument());
-        if (caseData.getManageOrders() != null && caseData.getManageOrders().getJudgeDirectionsToAdminAmendOrder() != null) {
+        if (!StringUtils.isEmpty(selectedOrder.getJudgeNotes())) {
```

Comment:
Is there any chance selectedOrder can be null?

Reply:
No